worker-markup-core/src/main/java/com/github/cafdataprocessing/worker/markup/core/XmlConverter.java

```diff
@@ -161,7 +157,6 @@ private static void addEmailHeadersToXmlFieldEntries(List<XmlFieldEntry> sourceE
 
     private static void appendEmailHeaderToBuilder(StringBuilder headersBuilder, String headerStartText,
                                                    String headerSourceFieldName, List<XmlFieldEntry> sourceEntries)
-        throws AddHeadersException
     {
         List<String> headerValues = sourceEntries.stream()
                 .filter(ent -> ent.getName().toLowerCase(Locale.ENGLISH).equals(headerSourceFieldName))
```

Comment:
It wouldn't have mattered before, since we didn't accept multiple values, but now that we do we're going to need to sort the values to make sure the order is consistent.

Reply:
This problem needs to be addressed before hashing the values as we cannot guarantee the order the header values extracted from CONTENT of email replies for initial email. JIRA raised as [CAF-3833](https://jira.autonomy.com/browse/CAF-3833).